packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -46,6 +48,12 @@ Okta serves pages on your custom domain over HTTPS. To set up this feature, you
 
 You can also [configure a custom email address](#about-custom-email-addresses) to present a branded experience to your end users.
 
+### Multibrand and custom domains <ApiLifecycle access="ea" />
+
+You can create up to three custom domains with multibrand customizations and up to 200 custom domains by contacting support to increase your limit.
+
+You can only visit a branded touchpoint (such as a logo, color, favicon, or an image applied to a page or email) once you map to a custom domain. After you create a brand, map it to a custom domain. Then you can make further customizations, preview them, and publish them.
```

Comment:
You can only preview or visit a branded page (such as viewing brand assets applied to the Okta-hosted sign-in page) once you map....

Reply:
done.